packages/cozy-interapp/README.md

```diff
@@ -22,3 +39,87 @@ intents.create('io.cozy.apps', 'CREATE')
     // }
   })
 ```
+
+Example to use `removeIntentFrame()` method (by passing the flag `exposeIntentFrameRemoval` flag):
+```js

```

Comment:
Line break is missing before code block, GitHub supports this but I am not sure that mkdocs will like it.